### DISCUSSION TITLE
Feature request Support control thrust coefficient?

### DISCUSSION THREAD

**Discussion by Halation666** [NatLabRockies/floris#560] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Hello,I'm a beginner in wind farm control.First of all, I would like to say that the Floris model is excellent.
But now I want to control both the yaw angle and thrust coefficient(fct).I don't know if the current V3 model supports it.From your documents, I only see how to change the yaw angle for simulation.

**Describe the solution you'd like**
If this model supports controling the thrust coefficient, can you tell me the implementation method?Thanks a lot!!!

**Describe alternatives you've considered**

**Additional context**

**Comment by paulf81** [comment, marked as answer]
Dear @Halation666 , I apologize for only now seeing this discussion.  There is a related pull request but it is not on the near term agenda for merge:
PR #477 

If you are still interested in this feature, would you mind to submit an issue "feature enhancement" request?
